datashuttle/tui/screens/modal_dialogs.py

```diff
@@ -183,6 +205,27 @@ def compose(self) -> ComposeResult:
             id="select_directory_tree_container",
         )
 
+    @staticmethod
+    def get_drives():
+        if platform.system() == "Windows":
+            return [disk.device for disk in psutil.disk_partitions(all=False)]
+
+        elif platform.system() in ["Darwin", "Linux"]:
+            return ["/"] + [
+                f"/{d}" for d in os.listdir("/") if os.path.isdir(f"/{d}")
```

Comment:
could rename `d` to `dir` just for readability

Reply:
also even though it's slightly more verbose, it makes sense to use `pathlib` here for consistency with the rest of the codebase e.g.

```
f"/{dir.name}" for dir in Path("/").iterdir() if dir.is_dir()
```